dynamodb/dynamo_lock_table.go

```diff
@@ -15,6 +16,26 @@ import (
 // running many automated tests in parallel, we use a counting semaphore
 var tableCreateDeleteSemaphore = NewCountingSemaphore(10)
 
+// Terraform requires the DynamoDB table to have a primary key with this name
+const ATTR_LOCK_ID = "LockID"
```

Comment:
What are the implications of this with multiple Terraform/Terragrunt modules each writing a lock? Does that mean that otherwise parallelizable Terraform modules can no longer be run in parallel?